utils/lsvols

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+#
+# listvols:  Show all volumes, along with any cluster/worker mappings if any
+#
+# ToDo:  region as command arg
+#
+
+import sys
+import json
+import urllib.request
+from pprint import pprint as pp
+from os.path import expanduser
+import SoftLayer
+
+home = expanduser("~")
+bmxconfig = home + "/.bluemix/config.json"
+
+with open(bmxconfig) as f:
+    data = json.load(f)
+
+token= data['IAMToken']
+
+Bmgr = SoftLayer.BlockStorageManager(SoftLayer.Client())
+
+def get_regions():
+      regions = []
+      hdrs={'Content-Type': 'application/json', 'Authorization' : token }
+      try:
+           request = urllib.request.Request(url="https://containers.bluemix.net/v1/regions", headers=hdrs)
+           contents = json.load(urllib.request.urlopen(request))
+      except:
+           print ("Can't access Cloud API.  Are you logged in?  ")
+           sys.exit(-1)
+      for c in contents['regions']:
+          regions.append(c['name'])
+      return regions
+
+def get_volume_access (volid):
+      lba = Bmgr.get_block_volume_access_list(volid)
+      if len(lba['allowedIpAddresses']) >= 1:
+          return (lba['allowedIpAddresses'][0]['allowedHost']['name'],
+             lba['allowedIpAddresses'][0]['ipAddress'])
```

Comment:
The return is only giving back the first IP address details assuming the volume is authorized to a single host. It may have multiple host authorizations as well and hence you can return an array of allowed IPs and find if the worker IP is found in that list.